tests/CliBundle/Command/ImportMeshUniverseCommandTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+namespace Tests\CliBundle\Command;
+
+use Ilios\CliBundle\Command\ImportMeshUniverseCommand;
+use Ilios\MeSH\Model\DescriptorSet;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Mockery as m;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class ImportMeshUniverseCommandTest
+ * @package Tests\CliBundle\Command
+ */
+class ImportMeshUniverseCommandTest extends TestCase
+{
+    use m\Adapter\Phpunit\MockeryPHPUnitIntegration;
+
+    /**
+     * @var m\MockInterface
+     */
+
+    protected $meshParser;
+
+    /**
+     * @var m\MockInterface
+     */
+    protected $descriptorManager;
+
+    /**
+     * @var CommandTester
+     */
+    protected $commandTester;
+
+    /**
+     * @var string
+     */
+    const COMMAND_NAME = 'ilios:maintenance:import-mesh-universe';
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp()
+    {
+        $this->meshParser = m::mock('Ilios\MeSH\Parser');
+        $this->descriptorManager = m::mock('Ilios\CoreBundle\Entity\Manager\MeshDescriptorManager');
```

Comment:
Should be m::mock(Parser::class); - makes refactoring much easier as you can look for references and not strings.